packages/pds-ember/app/styles/pds/components/textarea/utils.scss

```diff
@@ -0,0 +1,5 @@
+@forward "__config" show
```

Comment:
Do you like how this ended up? (seems really nice to me)

Reply:
It seems to work, so I'm inclined to keep using this pattern.